config/locales/candidate_interface/contact_details.yml

```diff
@@ -50,14 +50,19 @@ en:
               too_long: Phone number must be %{count} characters or fewer
             address_line1:
               blank: Enter your building and street
+              international_blank: Enter your address
               too_long: Building and street line 1 must be %{count} characters or fewer
+              international_too_long: Address line 1 must be 50 characters or fewer
```

Comment:
Could you use `%{count}` in these new strings to avoid the hard-coded `50`s

Reply:
Yep, I think `%{count}` is given to us by max option. So I've added count argument to get the same behaviour 👍